Packs/BmcITSM/Integrations/BmcITSM/BmcITSM.yml

```diff
@@ -0,0 +1,2860 @@
+      - Under Investigation
+      - Scheduled For Correction
+      - Assigned To Vendor
+      - No Action Planned
+      - Corrected
+      - All
+  - name: ticket_impact
+    display: Ticket Impact
+    required: false
+    defaultvalue: All
+    type: 16
+    additionalinfo: The impact of the tickets to fetch.
+    options:
+      - 1-Extensive/Widespread
+      - 2-Significant/Large
+      - 3-Moderate/Limited
+      - 4-Minor/Localized
+      - All
+  - name: ticket_urgency
+    display: Ticket Urgency
+    required: false
+    defaultvalue: All
+    type: 16
+    additionalinfo: The urgnecy of the tickets to fetch.
+    options:
+      - 1-Critical
+      - 2-High
+      - 3-Medium
+      - 4-Low
+      - All
+  - name: query
+    display: Fetch By Query
+    required: false
+    defaultvalue:
+    type: 0
+    additionalinfo: 'Search qualification to fetch tickets. For example: Status = "Draft" AND Impact = "1-Extensive/Widespread". The query will be used as an addition to the existing params. You can Review the BMC documentation for how to Build search qualifications: https://docs.bmc.com/docs/ars2008/building-qualifications-and-expressions-929630007.html'
+  - display: Fetch incidents
+    name: isFetch
+    required: false
+    type: 8
+  - additionalinfo: "Choose the direction to mirror the incident: Incoming (from  BMC Helix ITSM to XSOAR), Outgoing (from XSOAR to BMC Helix ITSM), or Incoming and Outgoing (from/to XSOAR and BMC Helix ITSM)."
+    defaultvalue: None
+    display: Incident Mirroring Direction
+    name: mirror_direction
+    options:
+      - None
+      - Incoming
+      - Outgoing
+      - Incoming And Outgoing
+    required: false
+    type: 15
+  - additionalinfo: When selected, closing the BMC Helix ITSM ticket is mirrored in Cortex XSOAR.
+    defaultvalue: "false"
+    display: Close Mirrored XSOAR Incident
+    name: close_incident
+    required: false
+    type: 8
+  - additionalinfo: When selected, closing the XSOAR incident is mirrored in BMC Helix ITSM.
+    defaultvalue: "false"
+    display: Close Mirrored BMC Helix ITSM Ticket
+    name: close_ticket
+    required: false
+    type: 8
+  - name: proxy
+    display: Use system proxy settings
+    required: false
+    defaultvalue: "false"
+    type: 8
+    additionalinfo:
+  - name: insecure
+    display: Trust any certificate (not secure)
+    required: false
+    defaultvalue: "false"
+    type: 8
+    additionalinfo:
+  - display: Incident type
+    name: incidentType
+    required: false
+    type: 13
+script:
+  script: ""
+  type: python
+  subtype: python3
+  dockerimage: demisto/python3:3.10.5.31928
+  commands:
+    - name: bmc-itsm-user-list
+      description: List users profile in BMC Helix ITSM. The records are retrieved according to a constraint determined either by the query argument or by the filtering arguments. When using filtering arguments, Each one defines a 'LIKE' operation and 'AND' operator is used between them.
```

Comment:
Recommend the following:
List of user profiles in BMC Helix ITSM. The records are retrieved by the query argument or by the filtering arguments. When using filtering arguments, Each one defines a 'LIKE' operation, and the 'AND' operator is used between them.